composer.json

```diff
@@ -28,68 +28,80 @@
     "docs": "https://pimcore.com/docs/latest/"
   },
   "require": {
-    "php": "~8.1.0 || ~8.2.0",
+    "composer-runtime-api": "^2.0",
+    "composer/ca-bundle": "^1.2",
+    "defuse/php-encryption": "^2.2.0",
+    "doctrine/annotations": "^1.13.1 || ^2.0",
+    "doctrine/collections": "^1.6.5",
+    "doctrine/dbal": "^3.5",
+    "doctrine/doctrine-bundle": "^2.6.3",
+    "doctrine/doctrine-migrations-bundle": "^3.2.2",
+    "doctrine/inflector": "^2.0.2",
+    "doctrine/persistence": "^3.0.3",
+    "egulias/email-validator": "^3.0.0",
+    "endroid/qr-code": "^4",
     "ext-SimpleXML": "*",
+    "ext-curl": "*",
     "ext-dom": "*",
     "ext-exif": "*",
     "ext-fileinfo": "*",
     "ext-gd": "*",
-    "ext-intl": "*",
     "ext-iconv": "*",
+    "ext-intl": "*",
     "ext-json": "*",
     "ext-mbstring": "*",
     "ext-pdo": "*",
     "ext-pdo_mysql": "*",
     "ext-zip": "*",
     "ext-zlib": "*",
```

Comment:
I think these (including `php`) should be on the top (with `php` being the first).